src/components/HomePage/Banner.js

```diff
@@ -211,47 +185,80 @@ function Banner() {
-				>
-					<Grid item sm={12} md={6}>
-						<Grid
-							container
-							direction='column'
-							justify='flex-start'
-							alignItems='flex-start'
+	function renderInfo() {
+		const tz = getTimeZoneWithFormat(hothStart, 'short');
+		const month = monthFormatter.format(hothStart);
+		const startDay = hothStart.getDate();
+		const endDay = hothEnd.getDate();
+		const eventCrossesDate = startDay !== endDay;
+		const endDayString = eventCrossesDate ? `–${endDay}` : '';
+
+		return (
+			<>
+				<Typography component='h1' variant='h1' className={classes.text} style={{
+					fontWeight: 'bold'
+				}}>
+                    HOTH <span className={classes.hothNumber}>X</span>
+				</Typography>
+
+				<NoSsr>
+					<Countdown
+						date={hothStart}
+						renderer={countdownRenderer}
+					/>
+				</NoSsr>
+
+				<Typography variant='h5' className={classes.text} component='h3'
+					style={{ fontWeight: 500, marginTop: 10 }}><b>Location:</b> Carnesale Commons Palisades Room
+				</Typography>
+
+				<Box display='flex' alignItems='left' color='white'>
+					<Tooltip
+						title={`This date range is displayed in your timezone! (${tz})`}
+						placement='top'
+						arrow={true}
+					>
+						<Typography
+							variant='h5'
+							className={classes.text}
+							style={{ marginBottom: 10, fontWeight: 500 }}
+							component='h3'
 						>
-							{renderInfo(classes)}
-							{smallScreen ?
-								<NoSsr>
-									<Countdown
-										date={hothStart}
-										renderer={countdownRenderer}
-									/>
-								</NoSsr> :
-								null}
-						</Grid>
+							<time dateTime={hothStart.toISOString()} hidden>
+								{month} {startDay}{endDayString}, 2022
+							</time>
+							<b>Date:</b> Sunday, March 5, 2023
+						</Typography>
+					</Tooltip>
+				</Box>
+
+				{Date.now() > applicationOpen.getTime() && Date.now() < applyDeadline.getTime() &&
+                    <Button className={classes.apply} href={'https://forms.gle/4zSBeQh3cjRtqcg79'} target='_blank'>
+                        Apply Now
+                    </Button>
+				}
+			</>
+		);
+	}
+
+	return (
+		<div className={classes.background}>
+			<Container maxWidth='md' className={classes.container}>
+				<Box item sm={12} md={6} className={classes.info}>
+					<Grid
+						container
+						direction='column'
+						className={classes.grid}
+					>
+						{renderInfo()}
 					</Grid>
-					{smallScreen ?
-						null :
-						<Grid item sm={12} md={6}>
-							<Box display='flex' flexDirection='column'>
-								<NoSsr>
-									<Countdown
-										date={hothStart}
-										renderer={countdownRenderer}
-									/>
-								</NoSsr>
-								<Box><Img fluid={data.placeholderImage.childImageSharp.fluid} /></Box>
-							</Box>
-						</Grid>
-					}
-				</Grid>
+				</Box>
+				{smallScreen ?
+					null :
+					<Box display='flex' flexDirection='column' className={classes.img}>
+						<Box><Img fluid={data.placeholderImage.childImageSharp.fluid} /></Box>
```

Comment:
What is this <Box> for again? Was it so that the flexbox treats it as an item?

Reply:
The inner box around the `<img>` is unnecessary, I think. I believe that was from when we had the count down in the outer box. I think we can delete it.

Reply:
Deleted!